validation/views.py

```diff
@@ -380,6 +382,23 @@ def register(request):
 # TODO: Speaker bio page like https://ojibwe.lib.umn.edu/about/voices
 
 
+@require_http_methods(["POST"])
+def record_translation_judgement(request, phrase_id):
+    # TODO: check that user is logged in
```

Comment:
Will `@login_required` help here?

https://docs.djangoproject.com/en/3.1/topics/auth/default/#the-login-required-decorator

Reply:
That does look useful, but I'll address this in its own PR.